.github/workflows/ci.yaml

```diff
@@ -28,6 +28,12 @@ jobs:
           - 7.4
           - 7.3
         include:
+          - description: Tests - Symfony 6.0
+            php: 8.0
+            symfony: '6.0.*'
```

Comment:
This is unneeded: the "normal" jobs will already test against the latest Symfony version, I was asking you to add 5.4 to avoid skipping it.
```suggestion
```